### ISSUE TITLE
F-MMF2-ECPP build is broken

### ISSUE THREAD

**Issue by brhillman** [E3SM-Project/E3SM#3982]
Builds with ECPP appear to be broken. Build error is as follows:
```
/home/bhillma/codes/e3sm/branches/master/components/eam/src/physics/crm/ecpp/ecpp_modal_wetscav.F90:1340:34:
  use modal_aero_wateruptake, only:  modal_aero_kohler
                                  1
Error: Symbol 'modal_aero_kohler' referenced at (1) not found in module 'modal_aero_wateruptake'
```
I suspect this is due to recent changes in the `modal_aero_wateruptake` code, and was able to break this case because we do not actually run the ECPP test as part of the development or integration suites. To reproduce this failure, all that is needed is to try to run a test case with this compset:
```
./create_test ERS_Ln9_P32x1.ne4pg2_ne4pg2.F-MMF2-ECPP
```

**Comment by whannah1** [comment]
Proposed solution = stop testing this until we get back to rewriting ECPP and hooking it up with the implementation of P3 that @pressel will start on soon.

**Comment by brhillman** [comment]
It's not in the development or integration suites anyways.

**Comment by whannah1** [comment]
This was fixed by PR #4145.
